composer.json

```diff
@@ -1,10 +1,20 @@
 {
-    "name": "lycheeorg/lychee-laravel",
+    "name": "lycheeorg/lychee",
```

Comment:
Just something which came to my mind again. The Github URL is `LycheeOrg/Lychee` with three capital letters. I don't know if this is relevant for anything at all or in case "something" interprets this entry, that "something" does so in a case-sensitive manner.

Anyway, I just re-remembered that we are astonishingly inconsistent about our capitalization: not only here, but also in the frontend, etc.

Reply:
I would go for `lychee-org/lychee` because our other packages on [packagist.org](https://packagist.org/?query=lychee) are under that common name

Reply:
It's fairly common for package names etc to be lower case while the project itself has capitalisation. By no means universal, but common. Especially with package managers.

Reply:
I only just reread this. **Why** the "-"??? (@ildyria)

Reply:
No clue. Just matching.

Reply:
I honestly don't remember, the only packages that are uploaded on packagist anyway are dependencies.
I guess my intent was because Capital letters are not "allowed" in package names and thus I wanted to avoid lycheeorg (which is ugly). But don't quote me on that.

Reply:
Yes, @ildyria was. That's another inconsistency we have and when I spotted it, it had been to late.

I guess we all agree that it would be awesome to use the same spelling of "Lychee Org" everywhere, i.e. here on Github and on [Packagist](https://packagist.org/). I don't know if it is possible to rename our packages and create some kind of "redirection" on Packagist from the old name to the new name such that we don't break any dependencies.

Reply:
```
    "require": {
        "php": "^8.0",
        "ext-bcmath": "*",
        "ext-ctype": "*",
        "ext-exif": "*",
        "ext-fileinfo": "*",
        "ext-gd": "*",
        "ext-json": "*",
        "ext-mbstring": "*",
        "ext-openssl": "*",
        "ext-pdo": "*",
        "ext-tokenizer": "*",
        "ext-xml": "*",
        "bepsvpt/secure-headers": "^7.1",
        "darkghosthunter/larapass": "dev-LycheeSpecial",
        "doctrine/dbal": "^3.1",
        "fideloper/proxy": "^4.3",
        "geocoder-php/cache-provider": "^4.3",
        "geocoder-php/nominatim-provider": "^5.5",
        "laravel/framework": "^8.83.14",
        "livewire/livewire": "^2.7",
        "lychee-org/nestedset": "^6",
        "lychee-org/php-exif": "^0.7.11",
        "maennchen/zipstream-php": "^2.1",
        "php-ffmpeg/php-ffmpeg": "^1.0",
        "php-http/guzzle7-adapter": "^1.0",
        "php-http/message": "^1.12",
        "spatie/guzzle-rate-limiter-middleware": "^2.0",
        "spatie/laravel-feed": "^4.0",
        "spatie/laravel-image-optimizer": "^1.6.2",
        "symfony/cache": "^v6.0.0",
        "whichbrowser/parser": "^2.0"
    },
    "require-dev": {
        "ext-imagick": "*",
        "ext-posix": "*",
        "ext-zip": "*",
        "barryvdh/laravel-debugbar": "^3.6",
        "barryvdh/laravel-ide-helper": "^2.10",
        "filp/whoops": "^2.5",
        "friendsofphp/php-cs-fixer": "^3.3",
        "itsgoingd/clockwork": "^5.0",
        "laravel/homestead": "^v13.2.1",
        "lychee-org/phpstan-lychee": "dev-master",
        "mockery/mockery": "^1.5",
        "nunomaduro/collision": "^5.0",
        "nunomaduro/larastan": "^1.0",
        "php-parallel-lint/php-parallel-lint": "^1.3",
        "phpunit/phpunit": "^9"
    },
```
camelCase does not seem standard there.

Reply:
If we take example on https://github.com/FriendsOfPHP it _should_ be "lycheeorg"

Reply:
I was just curious :). I'm slightly in favour of `lycheeorg` but it's not a major issue for me.

FWIW, we're `lycheeorg` on Docker Hub. That one was probably me.

Reply:
Found in a SO post: https://github.com/composer/packagist/issues/47